Cargo.toml

```diff
@@ -8,4 +8,18 @@ license = "Apache-2.0"
 repository = "https://github.com/westnordost/country-boundaries-rust/"
 categories = ["internationalization"]
 keywords = ["geocoding", "openstreetmap"]
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = []
+
+# When using any of these features, you have to comply with the terms of the ODbl license.
+# You must mention that the data your code uses is attributed to OpenStreetMap contributors:
+#
+#     The source data is licensed (c) OpenStreetMap contributors, OpenDatabase License 1.0
+#
+with_ODBL_licensed_OSM_data_low = ["lazy_static"]
+with_ODBL_licensed_OSM_data_mid = ["lazy_static"]
+with_ODBL_licensed_OSM_data_high = ["lazy_static"]
```

Comment:
renamed. There is almost no point in NOT exposing it -- unlike other systems, if something is not exposed on the crate itself (e.g. as inlined data blobs under different features), it is not available to the user code -- you can't link to it AFAIK. So the only reason to have anything like this data without exposing it is for testing, benchmarkig, examples, documentation - but not for anything that can actually be used in the user's code.

Reply:
Yeah, I mean, I have no issue with removing these other files from the repository. I think that is actually for the better, they are just copies from the other countryboundaries repository anyway. And it would be possible to generate any number of these files, with different raster sizes.